app/uk/gov/hmrc/customs/declaration/model/configModels.scala

```diff
@@ -16,8 +16,11 @@
 
 package uk.gov.hmrc.customs.declaration.model
 
-trait DeclarationsConfig {
-  val apiSubscriptionFieldsBaseUrl: String
-  val customsNotificationBaseBaseUrl: String
-  val customsNotificationBearerToken: String
-}
+case class DeclarationsConfig (apiSubscriptionFieldsBaseUrl: String,
+                               customsNotificationBaseBaseUrl: String,
+                               customsNotificationBearerToken: String)
+
+case class DeclarationsCircuitBreakerConfig(numberOfCallsToTriggerStateChange: Int,
```

Comment:
Circuitbreaker config should be per service based instead one for all.

Reply:
Not possible due to endpoint versions.